src/test/java/com/puppycrawl/tools/checkstyle/grammar/comments/AllBlockCommentsTest.java

```diff
@@ -48,8 +51,16 @@ protected String getPackageLocation() {
     public void testAllBlockComments() throws Exception {
         final DefaultConfiguration checkConfig =
                 createModuleConfig(BlockCommentListenerCheck.class);
+        final String path = getPath("InputFullOfBlockComments.java");
+        if (new FileText(new File(path), StandardCharsets.UTF_8.name())
+                .getFullText().chars().anyMatch(character -> character == '\r')) {
+            lineSeparator = "\r\n";
+        }
+        else {
+            lineSeparator = "\n";
+        }
```

Comment:
sounds like util method to be reused.

Reply:
Indeed, refactored and replaced all four occurrences.